test/cmd/builds.sh

```diff
@@ -91,21 +91,21 @@ frombuild=$(oc start-build --from-build="${started}")
 os::cmd::expect_success_and_text "oc describe build ${frombuild}" 'centos/ruby-22-centos7$'
 echo "start-build: ok"
 
-os::cmd::expect_success "oc cancel-build ${started} --dump-logs --restart | grep 'Restarted build ${started}.'"
+os::cmd::expect_success_and_text "oc cancel-build ${started} --dump-logs --restart" "Restarted build ${started}."
 os::cmd::expect_success 'oc delete all --all'
 os::cmd::expect_success 'oc process -f examples/sample-app/application-template-dockerbuild.json -l build=docker | oc create -f -'
-os::cmd::expect_success "tryuntil 'oc get build/ruby-sample-build-1'"
+os::cmd::try_until_success 'oc get build/ruby-sample-build-1'
 # Uses type/name resource syntax to cancel the build and check for proper message
-os::cmd::expect_success "oc cancel-build build/ruby-sample-build-1 | grep 'Build build/ruby-sample-build-1 was cancelled.'"
+os::cmd::expect_success_and_text 'oc cancel-build build/ruby-sample-build-1' 'Build build/ruby-sample-build-1 was cancelled.'
 # Make sure canceling already cancelled build returns proper message
-os::cmd::expect_success "tryuntil $(oc cancel-build build/ruby-sample-build-1 | grep 'A cancellation event was already triggered for the build build/ruby-sample-build-1.')"
```

Comment:
I had to change the `grep` string here for some reason I can't quite understand - the output from the API server is:
```
FAILURE: oc cancel-build build/ruby-sample-build-1: the output content test failed
Standard output from the command:
A cancellation event was already triggered for the build ruby-sample-build-1.
A build can be cancelled only if it has new/pending/running status.
A build can be cancelled only if it has new/pending/running status.
A build can be cancelled only if it has new/pending/running status.
```

The build is not named with the full `build/buildname` syntax.